apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.$projectParam.triggers_.external.$triggerParam_.runs.$runParam/route.tsx

```diff
@@ -124,6 +126,7 @@ export default function Page() {
           id: trigger.id,
         }),
       }}
+      currUser={user}
```

Comment:
We prefer to write out the entire word, so can you rename this to `currentUser`?